### PR TITLE
✨ Feature/4862: MP Checks - MATSMTH-5, 6, 7

### PR DESCRIPTION
**Pull request by subrataroyCVP**
## [✨ Feature/4862: MP Checks - MATSMTH-5, 6, 7](https://app.zenhub.com/workspaces/emissioners-scrum-board-5f36cd263511ad001a777f8e/issues/gh/us-epa-camd/easey-ui/4862)

### CODE REVIEW

File: src/mats-method-workspace/mats-method-checks.service.spec.ts [hunk 1]
```diff
@@ -65,4 +65,19 @@ describe('Mats Method Checks Service Test', () => {
...
   });
+
+  describe('MATSMTH-5 Checks', () => {
+    it('Should get [MATSMTH-5-A] error', async () => {
+      payload.beginDate = new Date('2023-02-28');
+      payload.beginHour = 5;
+      payload.endDate = new Date('2023-02-28');
+      payload.endHour = 1;
+
+      try {
```
Comment (billmiller13) [inline src/mats-method-workspace/mats-method-checks.service.spec.ts:96]: This test won't fail even if the validation is not working right and fails to throw an error for bad dates

**Review by billmiller13 — COMMENTED** [review]
See inline comment

File: src/mats-method-workspace/mats-method-checks.service.spec.ts (hunk 1 above)
Comment (subrataroyCVP) [reply src/mats-method-workspace/mats-method-checks.service.spec.ts:96]: This is just checks beginDate is after endDate or not. This is currently throwing error. I am getting coverage for those lines.

Comment (subrataroyCVP) [reply src/mats-method-workspace/mats-method-checks.service.spec.ts:96]: image

**Review by billmiller13 — APPROVED** [review]

**Pull request merged**